server/fleet/hosts.go

```diff
@@ -39,6 +39,15 @@ const (
 	OnlineIntervalBuffer = 60
 )
 
+func (s HostStatus) IsValid() bool {
```

Comment:
```suggestion
func (s HostStatus) IsFilterSupportedForBulkOps() bool {
```

Reply:
`IsValid` follows the pattern for other filter and could be used outside of bulk ops

Reply:
ok